ajax/libs/jquery.tablesorter/package.json

```diff
@@ -17,10 +17,7 @@
     "alphanumeric",
     "natural"
   ],
-  "licenses": [
-    "MIT",
-    "deprecated_GPL-2.0+"
-  ],
+  "licenses": "(MIT OR GPL-2.0+)",
```

Comment:
I think it doesn't really matter, if there is any inconvenience caused, please feel free to discuss with us, thanks!